udata_front/theme/gouvfr/assets/js/types.ts

```diff
@@ -114,26 +114,11 @@ export type Harvest = {
   backend: string;
 }
 
-export type NewDataset = Owned & {
-  title: string;
-  acronym: string;
-  archived: boolean;
-  description: string;
-  tags: Array<string> | null;
-  license: string;
-  frequency: string;
-  temporal_coverage: string;
-  frequency_date: Date | null;
-  page: string;
-  private: boolean;
-  quality?: Quality;
-  spatial: {
-    zones?: Array<string>;
-    granularity?: string;
-  } | null;
-};
+export type OwnedWithId = { organization: string, owner: never | null } | { organization: never | null, owner: string };
+
+export type NewDataset = Omit<BaseNewDataset, keyof OwnedWithId> & OwnedWithId;
 
-export type Dataset = NewDataset & {
+export type Dataset = BaseNewDataset & {
```

Comment:
This Dataset type should be removed and everything using it should use the Dataset from datafouv/components